my-portfolio/src/app/app.component.html

```diff
@@ -1,20 +1,10 @@
-<!--The content below is only a placeholder and can be replaced.-->
+<head>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.8.0/p5.js"></script>
```

Comment:
What is this used for?

Reply:
This is the p5 library, that has the codes for sketches.